src/easyscience/Objects/new_variable/parameter.py

```diff
@@ -49,6 +49,7 @@ def __init__(
         min: Optional[numbers.Number] = -np.inf,
         max: Optional[numbers.Number] = np.inf,
         fixed: Optional[bool] = False,
+        free: Optional[bool] = True,
```

Comment:
To me it seems like this is not used and I would delete it. Also, it is probably dangerous both to fixed and free since they are mutual exclusive

Reply:
Deleted